### DISCUSSION TITLE
Getting the current BinaryView object in GetInstructionInfo, GetInstructionText arch callbacks

### DISCUSSION THREAD

**Discussion by cfircohen** [Vector35/binaryninja-api#2328] [API] [answered]
Hi!

Is there a way to get the current `BinaryView` object associated with the `address` passed to `GetInstructionInfo` and `GetInstructionText` architecture callbacks?

The motivation for this is https://github.com/google/binja-hexagon/issues/2

Thanks

**Comment by plafosse** [comment, marked as answer]
#551 This is unfortunately a known limitation that we are working to remedy.
